### PR TITLE
[CoreBundle] Managers can be added in the workspace administration

### PR DESCRIPTION
**Pull request by ngodfraind**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no

Administrators management for workspaces in the workspace administration list.

### CODE REVIEW

**Comment by ngodfraind** [comment]
Updating this, don't bother with it yet.

**Comment by ngodfraind** [comment]
@Elorfin 

I still need to fix a test, but what do you think about it ?
It include the serializer / api improvements we discussed before.

File: main/core/API/Finder/Workspace.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                $qb->andWhere("UPPER(obj.{$filterName}) LIKE :{$filterName}");
+                $qb->setParameter($filterName, '%'.strtoupper($filterValue).'%');
+            } else {
+                // catch boolean
+                if ('true' === $filterValue || 'false' === $filterValue) {
+                    $filterValue = 'true' === $filterValue;
+                }
+
+                $qb->andWhere("obj.{$filterName} = :{$filterName}");
+                $qb->setParameter($filterName, $filterValue);
```
Comment (Elorfin) [inline main/core/API/Finder/Workspace.php]: IMO we can put this part of the query in the Finder...

File: main/core/Resources/modules/layout/modal/index.js [hunk 2]
```diff
@@ -5,17 +5,20 @@ import {MessageModal} from './components/message.jsx'
...
 export const MODAL_URL = 'MODAL_URL'
+export const MODAL_USER_PICKER = 'MODAL_USER_PICKER'
 
 const modals = {
   [MODAL_MESSAGE]: MessageModal,
   [MODAL_CONFIRM]: ConfirmModal,
   [MODAL_DELETE_CONFIRM]: DeleteConfirmModal,
-  [MODAL_URL]: UrlModal
+  [MODAL_URL]: UrlModal,
+  [MODAL_USER_PICKER]: UserPickerModal
```
Comment (Elorfin) [inline main/core/Resources/modules/layout/modal/index.js]: I don't know if we really need to register this modal in all pages. You can register it only for workspace management in the app constructor (see quizzes).

File: main/core/API/Finder/Workspace.php (hunk 1 above)
Comment (ngodfraind) [inline main/core/API/Finder/Workspace.php]: I don't think so. It's specific to the object.

**Pull request merged**